text/0060-tasks-api.md

```diff
@@ -568,6 +568,151 @@ New task types are also added for these operations. `indexCreation`, `indexUpdat
 }
 ```
 
+#### 9. Filtering task resources
+
+The `/tasks` and `index/{indexUid}/tasks` endpoints are filterable by type and status query parameters.
```

Comment:
I know this is not the purpose of this spec, but since we are thinking about adding filters to the `/tasks` resources and our task id is global, why not add a filter to filter by `uid=`/`index=` and remove the `index/{indexUid}/tasks`/`index/{indexUid}/tasks/{id}`? I suppose it could make the development on the core side easier since we will need to handle only two APIs, not four.

Reply:
I 100% agree with that! The indexes task routes basically do just that. That's legacy

Reply:
Same! We discussed this with the core team this morning: v0.28.0 should at least support the filter by index to remove the `/indexes/{uid}/tasks` in v0.28.0 as well. This would avoid a huge breaking in the future.